### ISSUE TITLE
Accessibility: JAWS doesn't announce sorting information

### ISSUE THREAD

**Issue by zhr1130** [react-bootstrap-table/react-bootstrap-table2#1637]
**Describe the bug**
JAWS doesn't announce sorting information at react-bootstrap-table.. It only announces enter while user select sort icon. Narrator announces sorting information.

**To Reproduce**
Go to storybook and search for "sort", any example can repro this issue.
https://react-bootstrap-table.github.io/react-bootstrap-table2/storybook/index.html?selectedKind=Bootstrap%204&selectedStory=Sort%20table%20with%20bootstrap%204&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
